lib/phlex/rails/layout.rb

```diff
@@ -60,5 +44,30 @@ def self.included(klass)
 
 			klass.extend(Interface)
 		end
+
+		def render(view_context, *args, **kwargs, &block)
+			if @_context
```

Comment:
If `@_context` is already set, we're already rendering and this should not be treated as an external call to render the layout. It should do the normal `render` behaviour instead.

Reply:
We should probably define a method for this, e.g.

```ruby
def rendering?
  !!@_context
end
```